controlplane/pkg/apis/plugins/connection_plugin.proto

```diff
@@ -2,7 +2,15 @@ syntax = "proto3";
 
 package plugins;
 
-import "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/connectioncontext/connectioncontext.proto";
```

Comment:
May be we name it just Connection?

Reply:
I also don't like ConnectionInfo naming, but it we name it Connection, we will get methods like `NewConnection(connection)`, `connection.getConnection()` and `connection.SetConnection(connection)`. It confuses me more. What about ConnectionWrapper?

Reply:
Renamed to ConnectionWrapper

Reply:
Protobuf generate GetLocalConnection(), GetRemoteConnection(), so I think it is ok to have it named as just a connection.

Reply:
We could reused it in NSM since we have interface to handle both of them.

Reply:
Created https://github.com/networkservicemesh/networkservicemesh/issues/1497